command.ts

```diff
@@ -396,6 +396,19 @@ export type RedisCommands = {
   // Cluster
   // cluster //
   // Server
+  acl_auth(username: string, passwd: string): Promise<Status>;
```

Comment:
Is it possible to overload an existing method instead of defining a new one?

```suggestion
  auth(password: string): Promise<Status>;
  auth(username, password: string): Promise<Status>;
```

Reply:
Overloaded existing auth method - https://github.com/denolib/deno-redis/pull/90/commits/b112a7c8843a518522ac89a2e24adec7f0c22ecc